data/sites.json

```diff
@@ -5,7 +5,7 @@
     "tagline": "Bitcoin & Crypto made simple",
     "description": "A uniquely easy way to buy, sell & store crypto in Australia. Enjoy safe & effortless trading today.",
     "website": "https://digitalsurge.com.au/",
-    "logo": "https://tradersunion.com/images/brokers/743.jpeg",
+    "logo": "https://australianfintech.com.au/wp-content/uploads/sites/7/2018/07/Digital-Surge.png",
```

Comment:
Lets put this in our own repo so we cant get blocked, or our app break if it changes etc https://github.com/dltxio/idem-mobile/tree/development/data

Reply:
Sure thing :)